src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2349,6 +2349,35 @@ public void TestUsingWithDeclaration()
             Assert.NotNull(us.Statement);
         }
 
+        [Fact]
+        public void TestUsingVarWithDeclaration()
+        {
+            var text = "using T a = b;";
+            var statement = this.ParseStatement(text);
+
+            Assert.NotNull(statement);
```

Comment:
I think these tests would be far more readable if you use the `UsingTree` helper. I think most of the existing tests are written in this style because they existed before that helper did.

Reply:
Done and pushed; have both versions for now.